test/framework/options.py

```diff
@@ -1536,6 +1536,88 @@ def test_robot(self):
             ec_regex = re.compile(r'^\s\*\s\[[xF ]\]\s%s' % os.path.join(test_ecs_path, ecfile), re.M)
             self.assertTrue(ec_regex.search(outtxt), "Pattern %s found in %s" % (ec_regex.pattern, outtxt))
 
+    def test_module_features(self):
+        """Test use of --module-family and --module-properties."""
+        # use toy-0.0.eb easyconfig file that comes with the tests
+        eb_file = os.path.join(os.path.dirname(__file__), 'easyconfigs', 'toy-0.0.eb')
+
+        # put lmodrc file in place (properties must be known)
+        lmodrc = os.path.join(self.test_prefix, 'lmodrc.lua')
+        lmodrc_txt = '\n'.join([
```

Comment:
worst misusage of join ever. put this in a multiline module constant (it's fixed/readinly text  anyway).